src/server/engine_shard_set.cc

```diff
@@ -29,6 +33,11 @@ ABSL_FLAG(string, spill_file_prefix, "",
           " associated with tiered storage. E.g,"
           "spill_file_prefix=/path/to/file-prefix");
 
+ABSL_FLAG(dfly::MemoryBytesFlag, max_file_size, dfly::MemoryBytesFlag{},
```

Comment:
I suggest prefixing this flag with a feature name, aka `tiered_file_max_size`.
or `spill_file_max_size` for consistency though I think we won't use `spill file` externally so it probably not a good name,